src/sbvr-api/sbvr-utils.coffee

```diff
@@ -662,6 +708,33 @@ exports.getAbstractSqlModel = getAbstractSqlModel = (request) ->
 	request.abstractSqlModel ?= abstractSqlModels[request.vocabulary]
 	return request.abstractSqlModel
 
+prepareEnv = (httpReq, odata) ->
+	if _.isArray(odata)
+		_.map(odata, _.partial(cloneReq, httpReq))
+	else cloneReq(httpReq, odata)
+
+cloneReq = (httpReq, odata) ->
+	req =
```

Comment:
This is a `major` change btw, so we'll want to end up with this being merged along with the 7.x PR - so it might be worth targeting that with this PR or making the `req` changes be separate (I'm not certain if they're a hard dependency of the side effect hooks).